pkg/broker/filter/filter_handler.go

```diff
@@ -408,25 +439,10 @@ func (h *Handler) handleDispatchToSubscriberRequest(ctx context.Context, trigger
 		h.logger.Warn("Failed to delete TTL.", zap.Error(err))
 	}
 
-	reportArgs := &ReportArgs{
-		ns:          trigger.Namespace,
-		trigger:     trigger.Name,
-		broker:      brokerName,
-		filterType:  triggerFilterAttribute(trigger.Spec.Filter, "type"),
-		requestType: "filter",
-	}
-
-	if request.TLS != nil {
-		reportArgs.requestScheme = "https"
-	} else {
-		reportArgs.requestScheme = "http"
-	}
-
 	subscriberURI := trigger.Status.SubscriberURI
 	if subscriberURI == nil {
 		// Record the event count.
 		writer.WriteHeader(http.StatusNotFound)
-		_ = h.reporter.ReportEventCount(reportArgs, http.StatusNotFound)
```

Comment:
Why stop recording this metric?

Reply:
This was discussed on the google doc, the idea is that it can be derived from the dispatch duration histogram (which implicitly has a total count)

Since we are instrumenting the handler with otel as well, there is the `http.server.request.duration` histogram as well, so rejected/invalid events can be found from the difference between the two